evm_loader/program/src/entrypoint.rs

```diff
@@ -332,7 +341,14 @@ fn do_call<'a>(
     info!(match exit_reason {
         ExitReason::Succeed(_) => {
             let (applies, logs) = executor.deconstruct();
-            backend.apply(applies, logs, false)?;
+            let logs2 = backend.apply(applies, logs, false)?;
```

Comment:
Why do you need return logs from `backend.apply`? We already have it in this place (line 343).

Reply:
@s-medvedev you can just do not pass them to apply, so them willn't move and we can use them. But what if apply will need some behaviour with logs in future?

Reply:
We will not need to process the logs inside the apply method if you solve this problem now. In this case, it is better not to leave garbage in the code so that the logs inside apply do not cause unnecessary questions.